apps/admin-gui/src/app/shared/components/sponsored-members-list/sponsored-members-list.component.html

```diff
@@ -60,6 +60,21 @@
               mat-icon-button>
               <mat-icon>more_horiz</mat-icon>
             </button>
+            <span
+              *ngIf="passwdResetAuth(sponsoredMember)"
+              [matTooltip]="'SHARED.COMPONENTS.SPONSORED_MEMBERS_LIST.PASSWORD_RESET' | translate"
+              [matTooltipPosition]="'above'"
+            >
+              <button
+                (mouseenter)="disableRouting = true"
+                (mouseleave)="disableRouting = routingStrategy"
+                class="dark-hover-list-item"
+                aria-label="sponsors of member"
+                (click)="resetPassword(sponsoredMember)"
+                mat-icon-button>
+                <mat-icon>restore</mat-icon>
```

Comment:
```suggestion
                <mat-icon>vpn_key</mat-icon>
```
for this we probably dont have the perfect icon but I think this would be a little bit better